prez/sparql/methods.py

```diff
@@ -184,21 +184,18 @@ async def sparql_construct(query: str, prez: str):
 
 
 @alru_cache(maxsize=128)
```

Comment:
Does it make sense to cache this function call when it's performing a side effect to the database over HTTP?

Imagine if the following updates are made:

- Update label with value "aaa"
- Update label with value "bbb"
- Update label with value "aaa" but this doesn't get posted to the database because it's been cached

Reply:
good catch thank you